Bandwidth.Iris.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
     <metadata>
         <id>Bandwidth.Iris</id>
-        <version>1.0.14</version>
+        <version>1.0.15</version>
```

Comment:
Technically I think this update is a minor (new functionality, no breaking changes right?) so this should be `1.1.0`

Reply:
Agree, minor version more than a patch